content/blog/sprint-review-2106.md

```diff
@@ -0,0 +1,47 @@
+Title: Wazo Platform 21.06 Released
+Date: 2021-05-10T16:00:00
+Author: The Wazo Authors
+Category: Wazo Platform
+Tags: wazo-platform, development
+Slug: release-review-2106
+Status: published
+
+Hello Wazo Platform community!
+
+Here is a short review of the Wazo Platform 21.06 release.
+
+## New features in this release
+
+* **Retention**: We added the possibility to configure the automatic deletion of CDRs and call recordings. For instance, this makes it possible to abide by European GDPR regulations.
```

Comment:
Was already possible to configure CDR  retention days before
What is new:
* configurable via REST API AND by tenant
* delete recording file